KIF Tests/AccessibilityIdentifierTests.m

```diff
@@ -47,12 +47,14 @@ - (void)testWaitingForAbscenceOfViewWithAccessibilityIdentifier
 
 - (void)testLongPressingViewWithAccessibilityIdentifier
 {
+  [tester tapViewWithAccessibilityIdentifier:@"idGreeting"];
```

Comment:
Some weird indentation going on here

Reply:
It is only in github for some reason guessing using tabs vs spaces or something. I am planning on adding some type of formatter in the future so it automates this